cmd/gandi.go

```diff
@@ -23,6 +23,7 @@ type cli struct {
 	Debug         bool             `kong:"short='d',help='Enable debug logging'"`
 	DryRun        bool             `kong:"help='Enable dry run mode'"`
 	APIKey        string           `kong:"env='GANDI_KEY',help='The Gandi LiveDNS API key (may be stored in the GANDI_KEY environment variable)'"`
+	Token         string           `kong:"env='GANDI_PERSONAL_ACCESS_TOKEN',help='The Gandi Personal Access Token (PAT) (may be stored in the GANDI_PERSONAL_ACCESS_TOKEN environment variable)'"`
```

Comment:
what about using a shorter `GANDI_PAT` (not sure if we could have both)

Reply:
Both is probably unnecessary. `GANDI_PERSONAL_ACCESS_TOKEN` is pretty descriptive ("PAT" isn't a well-known acronym), and matches what [lego does](https://go-acme.github.io/lego/dns/gandiv5/)